src/examples/mouse-api-basic/example-mouse-api-basic.log-entry.ts

```diff
@@ -0,0 +1,5 @@
+export declare class LogEntry {
```

Comment:
please define it in the same component file (it's required when we will export this example to the stackblitz)